packages/react-components/react-utilities/src/compose/isSlot.test.tsx

```diff
@@ -1,31 +1,37 @@
 import * as React from 'react';
-import { isResolvedShorthand } from './isResolvedShorthand';
+import { isSlot } from './isSlot';
+import { slot } from './slot';
 
-describe('isResolvedShorthand', () => {
+describe('isSlot', () => {
   it('resolves a string', () => {
```

Comment:
nit: 

```suggestion
  it('handles a string', () => {
```

Please also update all below